control-plane-operator/controllers/hostedcontrolplane/assets/olm/catalog-operator-deployment.yaml

```diff
@@ -34,7 +34,24 @@ spec:
         - name: kubeconfig
           secret:
             secretName: service-network-admin-kubeconfig
+        - name: oas-konnectivity-proxy-cert
+          secret:
+            secretName: konnectivity-client
       containers:
+        - name: socks5-proxy
+          image: {{ imageFor hostedClusterConfigOperator }}
```

Comment:
this has the wrong syntax, should be:
`{{ imageFor "hosted-cluster-config-operator" }}`
Also, I was hoping that you'd introduce a new image name for the socks5-proxy in main. It's ok if for now you set it to the value of the hostedClusterConfigOperator image.

Reply:
Thanks Cesar, I've updated the syntax! I can introduce a new image name if you want though!